cppimport/importer.py

```diff
@@ -65,11 +65,17 @@ def load_module(module_data):
 
 def is_build_needed(module_data):
```

Comment:
In my opinion this method should return `True` if a build is needed, or have another name - as it's now it's very confusing and doesn't make sense.
I fixed this here by inverting the return value and adopting the method's usages.

Reply:
Totally agreed! Thanks for making that clearer.